From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 5 Mar 2020 21:33:59 -0800
Subject: [PATCH] Add Makefile to make packaging ucm2 easier.

---
 Makefile | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..86f8d40
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,7 @@
+all:
+
+check:
+
+install:
+	mkdir -p $(DESTDIR)/usr/share/alsa/ucm2
+	cp -arv ucm2/* $(DESTDIR)/usr/share/alsa/ucm2/
